src/kernel/src/syscalls/output.rs

```diff
@@ -44,3 +46,9 @@ impl From<NonZeroI32> for SysOut {
         }
     }
 }
+
+impl From<Fd> for SysOut {
+    fn from(value: Fd) -> Self {
+        value.get().into()
+    }
+}
```

Comment:
Better to implement this at the same place as `Fd` so this file will not get polluted with every types in our project. This file should contains only implementations for the external types.

Reply:
Sure, ok.